src/Components/Coins/CoinsTab.js

```diff
@@ -13,24 +12,16 @@ export default function CoinsTab() {
   const isCoinsPurchaseWindowOpen = mountedComponentsDict.CoinsPurchaseWindow;
 
   return (
-    <GeneralTab>
-      <Card id="coinsTab">
-        <Typography variant={'subtitle2'}>
-          <Box m={0.9}>
-            {coinsTabTexts.coinsAmountPt1}
-            XX
-            {coinsTabTexts.coinsAmountPt2}
-          </Box>
-        </Typography>
-
-        <GeneralButton id="getMoreCoinsButton" onButtonClick={() => mountComponents(['CoinsPurchaseWindow'])}>
-          {coinsTabTexts.getMore}
-        </GeneralButton>
-
-        {isCoinsPurchaseWindowOpen && (
-          <CoinsPurchaseWindow unmountCoinsPurchaseWindow={() => unmountComponents(['CoinsPurchaseWindow'])} />
-        )}
-      </Card>
+    <GeneralTab id="coinsTab">
+      {coinsTabTexts.coinsAmountPt1}
+      XX
+      {coinsTabTexts.coinsAmountPt2}
+      <GeneralButton id="getMoreCoinsButton" onButtonClick={() => mountComponents(['CoinsPurchaseWindow'])}>
+        {coinsTabTexts.getMore}
+      </GeneralButton>
+      {isCoinsPurchaseWindowOpen && (
+        <CoinsPurchaseWindow unmountCoinsPurchaseWindow={() => unmountComponents(['CoinsPurchaseWindow'])} />
+      )}
```

Comment:
should windows be renderer from within tabs?
doesnt seems like it at all.
might also make the css way simpler and maintainable.

Reply:
This is what the next branch is all about.